server/api.js

```diff
@@ -41,8 +41,11 @@ const con = mysql.createConnection({
 });
 
 con.connect(function(err) {
-	if (err) throw err;
-	console.log("Connected!");
+	if (err) {
```

Comment:
This change is just making error logging a bit clearer and is unrelated to the dependency change.